reformat.sh

```diff
@@ -19,8 +19,9 @@
 echo "Checking version number"
 VER=$(clang-format --version|cut -f3 -d' '|cut -f1 -d'.')
 if [ $VER -ge 8 ];then
-  set -euxo pipefail
-find {.,aws-encryption-sdk-cpp}/{include,source,tests} .cbmc-batch examples -name '*.h' -or -name '*.c' -or -name '*.cpp' | xargs clang-format -i
+    set -euxo pipefail
+    find {.,aws-encryption-sdk-cpp}/{include,source,tests} .cbmc-batch examples -name '*.h' -or -name '*.c' -or -name '*.cpp' | xargs clang-format -i
```

Comment:
Should this just be simplified to `find . -name '*.h' -or -name '*.c' -or -name '*.cpp' | xargs clang-format -i`

Reply:
Yes, added